web_poet/testing/__init__.py

```diff
@@ -0,0 +1,23 @@
+import json
+import os
+from pathlib import Path
+from typing import Any, Iterable, Union
+
+from itemadapter import ItemAdapter
+
+from web_poet.serialization import SerializedDataFileStorage, serialize
+
+INPUT_DIR_NAME = "inputs"
+OUTPUT_FILE_NAME = "output.json"
+
+
+def save_fixture(
```

Comment:
All other __init__ files in web-poet are import-only - what do you think about moving it out of here?